backend/character/character.py

```diff
@@ -0,0 +1,120 @@
+''' Representation of a single character and its stat spread '''
+
+import math
+from dataclasses import dataclass
+from backend.character.stat import Stat, Stats, ProbabalisticStat
+from backend.character.jobs import Roles, Buffs
+
+@dataclass
+class CharacterStatSpread: #pylint: disable=too-many-instance-attributes
+    """
+    Value class representation of a stat spread
+    """
+    wd: float #pylint: disable=invalid-name
```

Comment:
remove all invalid name pylint comments -- i stole nono's .pylintrc and allowed these variable names